distributed/tests/test_worker.py

```diff
@@ -3446,6 +3446,9 @@ def get_data(self, comm, **kwargs):
         return super().get_data(comm, **kwargs)
 
 
+@pytest.mark.xfail(
+    reason="https://github.com/dask-contrib/dask-expr/issues/945", raises=UserWarning
+)
```

Comment:
Not too happy about xfailing this, as it's not testing anything dataframe related.
I'll see if I can rewrite it with a generic graph.

Reply:
Doesn't specifying the raises keyword mean it's only failed if that warning is raised but would hard fail otherwise, is that right?

Reply:
Can we just catch the warning? No need to fail the whole test

Reply:
ya, that's probably better, and actually, I think we can just remove this b/c in a later commit that was squashed into this, I just added the ignore into `pyproject.toml` (https://github.com/dask/distributed/pull/8560/commits/f71ea01ead8f7f3dd55839440aa2e0b1bfe20e34)

Reply:
Okay done.. https://github.com/dask/distributed/pull/8560/commits/df07359e1e4834b4859c3fd830539045f094a593, sorry for the confusion. :sweat_smile: